build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.0.4'
+        classpath 'com.android.tools.build:gradle:7.1.0'
```

Comment:
I think gradle itself should be updated as a group to prevent having to sync with every branch change

Reply:
i agree

Reply:
done